CHANGELOG.md

```diff
@@ -165,4 +165,7 @@
  - Fix 500 error when downloading usage statistics
  - Fix name of downloaded usage file
  - Allow organizational admin to see the download button in 'Statisctics on Users' and 'Statistics on Tempalte' accordions
-- Upgrade node-sass to 7, node to 12 and adjust eslint test to use Github default action
+ - Upgrade node-sass to 7, node to 12 and adjust eslint test to use Github default action
+
+## [3.0.4+portage-3.0.0] - 2021-12-09
+  - Update mysql installation process for the Github action
```

Comment:
We can improve this change by adding the entry to the top of the file. Something like:

```
## [Unreleased]

### Changed
 - Update mysql installation process for the Github action

```

Sorry for the confusion, I missed adding the changes for version 3.0.0 when I tagged the release.

I will add the changes shortly.

Reply:
Got it. I'll change this one to the Unreleased section